recipes/pkutils/LICENSE

```diff
@@ -0,0 +1,7 @@
+Copyright 2018 Matt Durrant (mdurrant@stanford.edu)
```

Comment:
Seems that this is not the origina LICENSE file. The author in PyPI says  Reuben Cummings